### ISSUE TITLE
Animation missing for driver door locked.

### ISSUE THREAD

**Issue by rt-2** [Open-GTO/sa-mp-fixes#46]
This is an enhancement.
Basically when you try to enter a locked vehicle as passenger, there is an animation, and there is not for the driver door.
Thank you,
rt-2

**Comment by ziggi** [comment]
The fix already exitsted here:
http://forum.sa-mp.com/showthread.php?t=574093
https://github.com/ziggi/samp-enterfix/
but this need more tests.

**Comment by rt-2** [comment]
Yeah, I have it installed on my server and it didn't work, not sure if I set it up wrong or there was conflicts. 
@Y-Less  was trying to find all the improvment and fixes possible to add here a couple weeks ago.
Today I thought of this one, I think this should be added here, what do you think?
Thank you,
rt-2

**Comment by rt-2** [comment]
The script I tested before might not be yours, I was going to test this script again to see if it is working or not.
If I put `<enterfix.inc>` before `<fixes.inv>`:
```
C:\Users\rt-2\Desktop\SAARP\SAMP\pawno\include\fixes.inc(5513) : fatal error 111: user error: _ALS_OnPlayerUpdate defined
```
If I put `<fixes.inc>` before `<enterfix.inv>`:
```
C:\Users\rt-2\Desktop\SAARP\SAMP\pawno\include\enterfix.inc(52) : fatal error 111: user error: _ALS_OnPlayerEnterVehicle defined
```
This is what happens when EVERY include WANTS to be right next after <a_samp> smh.
rt-2

**Comment by Y-Less** [comment]
enterfix seems to have two callbacks hooked - one chains normally, the other errors, I'm not sure why.  There is usually no reason to error on `_ALS_Callback` existing, it is designed to chain functions fine.  The only reason that fixes.inc does it is because it has so many different hooks at a very low level that chaining them would be next to impossible while ensuring consistency.  It is a bit of an arrogant "this include is more important than yours" approach I know.

**Comment by rt-2** [comment]
Yes I checked the code and saw it was designed to give an error when it was not the first fix.
I understand fixes.inc doing that, but with this other include doing the same thing, now I can't test.
Ultimately I will test this and if it works we'll just add it into the script I guess and this conflict will be resolved anyway.
rt-2

**Comment by ziggi** [comment]
Should be fixed now. Check [enterfix](https://github.com/ziggi/samp-enterfix) again.

**Comment by rt-2** [comment]
Yes it compiled thank you.
I will test if it works fine for me later today, I think this should be added in fixes.inc ultimately?
rt-2

**Comment by ziggi** [comment]
> I think this should be added in fixes.inc ultimately?

Absolutely.

**Comment by rt-2** [comment]
I normally test a lot of my new script in filterscript, this only works in the gamemode, but works.
rt-2

**Comment by ziggi** [comment]
I also forgot about SetVehicleParamsForPlayer.

**Comment by rt-2** [comment]
My only problem right now is that all filterscript will see vehicles with unlocked doors because `ENTER_GetVehicleParamsEx` is not called from filterscript. 
This will be fixed when this will be added to fixes.inc.
Can't wait,
rt-2

**Comment by rt-2** [comment]
Well after this
https://github.com/ziggi/samp-enterfix/commit/c787a173ea305b040784d51ef2809909b9bb965a
I see no reason not to add it. I figured you guys are busy enough with real work. I might as well do it.
It was 'a bit' longer than I thought to add this in the **proper** way.
See: https://github.com/Open-GTO/sa-mp-fixes/pull/64
Thank you,
rt-2
